app/routers/profile.py

```diff
@@ -14,9 +14,12 @@
 from app.internal.import_holidays import (get_holidays_from_file,
                                           save_holidays_to_db)
 from app.internal.privacy import PrivacyKinds
+from app.internal.showevent import get_upcoming_events
 
 PICTURE_EXTENSION = config.PICTURE_EXTENSION
 PICTURE_SIZE = config.AVATAR_SIZE
+FIVE_EVENTS = 5
+"""We are presenting up to five upcoming events on the profile page"""
```

Comment:
inline comment should not be triple quote but with "#".

Reply:
done